packages/itwin/ec3-widget/src/EC3Widget.ts

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import type { Localization, TranslationOptions } from "@itwin/core-common";
+import { IModelApp } from "@itwin/core-frontend";
+
+export interface EC3WidgetConfig {
+  // Provide custom localized strings to override default localized strings. Corresponding keys can be found in EC3Widget.json
+  localizationOverrides?: Map<string, string>;
+  localization?: Localization;
+}
+
+/** EC3Widget is use when the package is used as a dependency to another app.
```

Comment:
This comment does not provide much useful information. Of course it will be used if it is used.

Reply:
removed